src/service/http/nebula/c/charts2.js

```diff
@@ -346,6 +346,31 @@ export class Charts {
         // 'bar'=>column, 'horizontalBar' for normal bar
         this.displayBar = (chartId, json, keys, metrics, vertical) => {
             this.cls(chartId);
+            // process json string for histogram
+            let histJsonIdx = -1;
+            for (var i = 0; i < metrics.length; i++) {
+                if (metrics[i].endsWith('.HIST')) {
+                    histJsonIdx = i;
+                    break;
+                }
+            }
+            if (histJsonIdx != -1) {
+                let histJsonStr = JSON.parse(json[0][metrics[histJsonIdx]]);
+                let hists = histJsonStr["b"];
+                let dict = {};
+                let labels = [];
+                for (var i = 0; i < hists.length; i++) {
+                    let count = hists[i][2];
+                    if (count == 0) {
```

Comment:
we may want to keep 0? (display all 20 bars even some of them don't have value), because usually users just look at the bars before figuring out which bar is for which range, less confusion. 

In the future, we may have UI option to allow user removing 0 sized bars.

Reply:
sorry, just saw these 2 comments, will update before merge